synapseAnnotations/data/sageCommunity.json

```diff
@@ -519,6 +519,21 @@
         "value": "mtx",
         "description": "Matrix Market Exchange Format",
         "source": "https://math.nist.gov/MatrixMarket/formats.html#MMformat"
+      },
+      {
+        "value":"bfilt",
+        "description": "contains the final ATACSeq peaks after removing the encode blacklist regions.",
+        "source":"https://sites.google.com/site/anshulkundaje/projects/blacklists"
+      },
+      {
+        "value":"tagAlign",
+        "description":"Tag Alignment provides genomic mapping of short sequence tags.",
+        "source":"https://genome.ucsc.edu/FAQ/FAQformat.html#format15"
+      },
+      {
+        "value":"dup",
```

Comment:
I'm guessing SAM/BAM aren't adequate for these files?

Reply:
Also, another strange file path .dup (https://www.synapse.org/#!Synapse:syn18359507). This output is the reprocessing pipeline for PEC - so we will continue to produce files with these file paths.

Reply:
tagAlign is a specific form of a bed file, dup is a SAM or BAM file with additional notation for duplicate reads...Maybe the files could be more broadly characterized, but why not be more granular and point to the specific documentation?

Reply:
yeah that seems fine if they are different in a standardized way. my main concern was with bfilt